tests/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,208 @@
+        ceph_validate = True
+    else:
+        log.error(f"{pool_name} pool failed to get created at CEPH side")
+    pool_obj = (POOL.get(resource_name=""))
+    sample = pool_obj['items']
+    for item in sample:
+        if item.metadata.name == pool_name:
+            OCS_validate = True
+    if ceph_validate and OCS_validate:
+        log.info("Pool got created successfully from Ceph and OCS side")
+        return True
+    else:
+        return False
+
+def get_toolbox_pod_handler():
+    """
+    Used to get handler of toolbox_pod
+
+    :return:
+    """
+    config.load_kube_config()
+    v1 = client.CoreV1Api()
+    ret = v1.list_pod_for_all_namespaces(
+        watch=False,
+        label_selector='app=rook-ceph-tools'
+    )
+
+    for i in ret.items:
+        namespace = i.metadata.namespace
+        name = i.metadata.name
+        break
+    tool_box = pod.Pod(name, namespace)
+    return tool_box
+
+def validate_storageclass(sc_name):
+    """
+    Validate if storageClass is been created or not
+    """
+    sc_obj = SC.get(resource_name="")
+    sample = sc_obj['items']
+    for item in sample:
+        if item.metadata.name in sc_name:
+            log.info(f"StorageClass got created successfully")
+            return True
+    return False
+
+def create_secret_rbd(**kwargs):
+    """
+    This will create Secret file which will be used for creating StorageClass
+
+    :return:
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SECRET_RBD_YAML, **kwargs
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert SECRET.create(yaml_file=TEMP_YAML_FILE)
+    return True
+
+def get_client_admin_keyring():
+    """
+    This will fetch client admin keyring from Ceph
+
+    :return:
+    """
+    cmd = f"ceph auth get-key client.admin|base64"
+    tool_box = get_toolbox_pod_handler()
+    out, err, ret = tool_box.exec_command(cmd=cmd, timeout=20)
+    if out:
+        out = out.rstrip('\n')
+        secret_data = {}
+        secret_data['client_admin_key'] = out
+        assert create_secret_rbd(**secret_data)
+    else:
+        log.error(f"Failed to get client auth key")
+        return StatusOfTest.FAILED
+
+def create_pvc():
+    """
+    This will create PVC with default value
+
+    :return:
+    """
+
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        PVC_RBD_YAML,
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert PVC.create(yaml_file=TEMP_YAML_FILE)
+    log.info(f"Waiting for 3 sec for PVC to get created")
+    sleep(3)
+    return True
+
+def validate_pvc(pvc_name):
+    """
+    This will check if PVC created is bound successfully or not
+
+    :param pvc_name:
+    :return:
+    """
+    pvc_list = PVC.get(resource_name=pvc_name)
```

Comment:
when you provide a resource_name you can expect to get only one item and not a list